src/web/lib/components/SavedThemeSelector/index.js

```diff
@@ -21,7 +21,7 @@ export const SavedThemeSelector = ({
       themes={sortedSavedThemes}
       className="saved-theme-selector"
       previewClassName="saved-theme-preview"
-      onClick={theme => setTheme({ theme })}
+      onClick={theme => setTheme({ theme, userEdit: false })}
```

Comment:
Making `meta.userEdit` false here gets this edit filtered out of the undo stack, but I still see the Save button get active when selecting a saved theme